guides/common/modules/proc_invalidating-jwts-of-all-users.adoc

```diff
@@ -0,0 +1,36 @@
+[id="invalidating-jwts-of-all-users"]
+= Invalidating JWTs of all users
+
+You can invalidate all registration JSON Web Tokens of all users at once.
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-invalidating-jwts-of-all-usersvvv[].
+To use the API, see the xref:api-invalidating-jwts-of-all-users[].
+
+.Prerequisites
```

Comment:
we don't need view_users permissions for api or cli

Reply:
That depends on how we write the procedure. In some cases it can be beneficial to list users to find their IDs before they run the main command. In such case, they would need the view_users permission, wouldn't they?

I'll revisit this once I have the commands ready.

Reply:
yeah but `view_users` permission is not mandatory for API and CLI, so it shall not be a prerequisite. It is however mandatory for UI.